rj_common/include/rj_common/Field_Dimensions.hpp

```diff
@@ -276,3 +232,38 @@ struct Field_Dimensions {
 
     std::vector<Geometry2d::Line> _FieldBorders;
 };
+
+namespace rj_convert {
+
+template <>
+struct RosConverter<Field_Dimensions, Field_Dimensions::Msg> {
+    static Field_Dimensions::Msg to_ros(const Field_Dimensions& from) {
+        Field_Dimensions::Msg to;
+        to.length = from.Length();
+        to.width = from.Width();
+        to.border = from.Border();
+        to.line_width = from.LineWidth();
+        to.goal_width = from.GoalWidth();
+        to.goal_depth = from.GoalDepth();
+        to.goal_height = from.GoalHeight();
+        to.penalty_short_dist = from.PenaltyShortDist();
+        to.penalty_long_dist = from.PenaltyLongDist();
+        to.center_radius = from.CenterRadius();
+        to.center_diameter = from.CenterDiameter();
+        to.goal_flat = from.GoalFlat();
+        to.floor_length = from.FloorLength();
+        to.floor_width = from.FloorWidth();
+        return to;
```

Comment:
I'd prefer if you used the builder, just so that if we change message definitions there's a compiler error and we don't run into bugs where we forget to convert a field